src/sugar3/speech.py

```diff
@@ -278,7 +278,12 @@ def make_pipeline(self, command):
             self.stop_sound_device()
             del self._pipeline
 
-        self._pipeline = Gst.parse_launch(command)
+        try:
+            self._pipeline = Gst.parse_launch(command)
+        except:
```

Comment:
@ignaciouy, can you uninstall gstreamer espeak plugin and check what error do you receive?
Check on _init_voices method too.
I think we can assume Gstreamer is installed and working for this case. If not we can't import Gst and can't do Gst.init() neither.